### ISSUE TITLE
Sometimes a Tail instance is returned instead of value

### ISSUE THREAD

**Issue by dtinth** [6to5/6to5#718]
This happens in my tests, based on various `console.log`.

I am unable to isolate this problem yet, but [it happens](https://travis-ci.org/bemusic/bemuse/builds/49926583).

My suspicion is that `_tailCall` function doesn't work across compiled modules because of a different `Tail` instance. I'll try to isolate this problem when I have more time.

**Comment by sebmck** [comment]
In the meantime you can blacklist the `es6.tailCall` transformer:

```sh
$ 6to5 --blacklist es6.tailCall
```

```javascript
require("6to5").transform("code", { blacklist: ["es6.tailCall"] });
```

**Comment by sebmck** [comment]
Should be fixed as of 3.5.1, thanks!
